src/PerfView/PerfView.csproj

```diff
@@ -55,9 +55,11 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.6.0" />
```

Comment:
The DLLs for these packages are going to need to be packaged into PerfView as <EmbeddedResource/> entries so that they are present on client machines.

Reply:
Can you elaborate? Doesn't it happen automatically?

Reply:
The unpacking of reosurces happens automatically, but what gets packed as resources does not.  Here's an example of how files get packaged as resources: https://github.com/microsoft/perfview/blob/main/src/PerfView/PerfView.csproj#L261.  I think you'll want to do this for the DLLs in these packages that are required to make these flows work.

If you look at the bin directory, the DLLs will be present there, but if you copy PerfView to a separate directory and then run it, it will use the unpacked version in %appdata%, which will mean that the new dependencies won't be present, and I'd expect these new flows to fail.

Reply:
Ah, OK.

Reply:
I fixed this by using a variant of the .targets file we use in the Snapshot Debugger to create a self-contained EXE. It automatically embeds all the resolved reference assemblies.